galaxy.yml

```diff
@@ -16,49 +14,50 @@ version: 1.0.0
 readme: README.md
 
 # A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+# @nicks:irc/im.site#channel'
 authors:
-  - IBM Lab for SAP Solutions
-  - IBM Cloud for SAP
-  - IBM Consulting for SAP
```

Comment:
Most of the work is done by IBM in this collection

Reply:
what collection you are talking about sap.sap_operations or community.sap_operations?

Reply:
you will only contribute to this **community.sap_operations** and at the moment the work done in this collection is made by ibm why do you delete it then ;-) ?

Reply:
This is another question, where there is a misunderstanding I assume.

What is required is a public repo with sap.sap_operations collection content. But separate repo cannot be created in SAP Linux Lab org. So here we are.

Reply:
We have the need to merge Kirill's development with the current upstream development. Some rewriting was needed because not everything from the IBM-created code is supportable. Hence, we need to merge the two repos and create a joint roadmap to which **all** can contribute.